### ISSUE TITLE
Reusing transformation script

### ISSUE THREAD

**Issue by ivozandhuis** [netwerk-digitaal-erfgoed/LDWizard#17]
Could we introduce a function that helps the user to reuse a previously created transformation script on the same but updated csv-file?

**Comment by wouterbeek** [comment]
Good point, let's start with specifying this in the context of M1.  We can create implementation issues afterwards.

**Comment by wouterbeek** [comment]
Reusing transformations is relatively complex, since it requires that the exported transformation script is imported in exactly the same way (i.e., resulting in the exact same settings).

Since we support multiple transformation formats, it must be possible to export as RML, import in the same wizard, export as CoW, import in the same wizard, and export to the same RML as before.

Transformation script reuse has implications for the changeability of LD Wizard: users may try to reuse an older script in a newer wizard.

Transformation reuse is a relatively advanced feature.

**Comment by wouterbeek** [comment]
Consider supporting fewer formats for reuse than for export.  (Less complexity for the reuse feature.)
